structures/multistep.js

```diff
@@ -0,0 +1,12 @@
+const s = require('superstruct')
+
+const multistep_package = s.size(s.array(s.string()), 13)
+const multistep_address = s.size(s.array(s.string()), 11)
+
+const Multistep = s.size(s.array(s.string()), 27, Infinity)
+
+module.exports = {
+  multistep_address,
```

Comment:
Sorry but I don't see the relation between `multistep_package`, `multistep_address` and `Multistep`.